### DISCUSSION TITLE
PrimeNg panel menu

### DISCUSSION THREAD

**Discussion by skabhiyanta** [primefaces/community#660] [PrimeNG] [answered]
I am using p-panelmenu. As per the document there is a property "state" in MenuItem. How to pass value of state property in navigation?

**Comment by cetincakiroglu** [comment, marked as answer]
You can simply do this:

```
items = [
  ......
  {
    label: '',
    .....,
    state: 'your state'
  }
  ......
]
```

The anchors receive the state value.
